src/components/PassthroughForm.vue

```diff
@@ -21,6 +21,10 @@ export default {
     successRedirectViewName: {
       type: String,
       default: ''
+    },
+    successRedirectPath: {
```

Comment:
This provides another option for how to pass in a success redirect. It is necessary because the route name alone is not sufficient for redirecting to certain routes that require a route param, like `/proposals/:id`